src/manifests/test_manifest.py

```diff
@@ -31,6 +31,7 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
                 - with-less-security
               additional-cluster-configs:
                 - key : value
+              ci-group: 6
```

Comment:
Why hardcode here?

Reply:
Looks like an example of the schema

Reply:
This is the schema for the test manifest. All these are commented out. Basically showing the structure and format for the test manifest.